### ISSUE TITLE
graph-data-science plugin cannot be installed into Neo4j v5.26.0 in docker 

### ISSUE THREAD

**Issue by AnikinNN** [neo4j/neo4j#13563]
### Bug description
`graph-data-science` plugin cannot be installed into Neo4j in docker

**Neo4j Version:** 5.26.0
**Host Operating System:** Ubuntu 22.04
**API:** Docker

### Steps to reproduce
run the `docker-compose.yml` below
```yaml
  neo4j:
    image: neo4j:5.26-community
    restart: always
    environment:
      NEO4J_PLUGINS: '["graph-data-science"]'
    volumes:
      - neo4j_data:/data
```

### Expected behavior
`graph-data-science` installed like it was in 5.25

### Actual behavior
```
neo4j-1  | Fetching versions.json for Plugin 'graph-data-science' from https://graphdatascience.ninja/versions.json
neo4j-1  | Installing Plugin 'graph-data-science' from https://graphdatascience.ninja/neo4j-graph-data-science-2.13.1.jar to /var/lib/neo4j/plugins/graph-data-science.jar 
neo4j-1  | Applying default values for plugin graph-data-science to neo4j.conf
neo4j-1  | Skipping dbms.security.procedures.unrestricted for plugin graph-data-science because it is already set.
neo4j-1  | You may need to add gds.* to the dbms.security.procedures.unrestricted setting in your configuration file.
neo4j-1  | Changed password for user 'neo4j'. IMPORTANT: this change will only take effect if performed before the database is started for the first time.
neo4j-1  | 2024-12-12 13:24:55.279+0000 INFO  Logging config in use: File '/var/lib/neo4j/conf/user-logs.xml'
neo4j-1  | 2024-12-12 13:24:55.291+0000 INFO  Starting...
neo4j-1  | 2024-12-12 13:24:55.929+0000 INFO  This instance is ServerId{xxxxxxxx} (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)
neo4j-1  | 2024-12-12 13:24:56.715+0000 INFO  ======== Neo4j 5.26.0 ========
neo4j-1  | 2024-12-12 13:24:56.863+0000 INFO  GDS license file: null
neo4j-1  | 2024-12-12 13:24:56.863+0000 INFO  GDS license state: unlicensed
neo4j-1  | 2024-12-12 13:24:56.865+0000 INFO  Prometheus: disabled
neo4j-1  | 2024-12-12 13:24:56.865+0000 WARN  GDS metrics disabled
neo4j-1  | 2024-12-12 13:24:56.869+0000 INFO  Progress tracking: enabled
neo4j-1  | 2024-12-12 13:24:56.870+0000 INFO  Memory usage guard: minimum estimate
neo4j-1  | 2024-12-12 13:24:56.873+0000 INFO  Register GDS Memory Facade...
neo4j-1  | 2024-12-12 13:24:56.874+0000 INFO  GDS Memory Facade registered.
neo4j-1  | 2024-12-12 13:24:56.881+0000 ERROR Failed to start Neo4j on 0.0.0.0:7474.
neo4j-1  | java.lang.RuntimeException: Error starting Neo4j database server at /data/databases
neo4j-1  |      at org.neo4j.graphdb.facade.DatabaseManagementServiceFactory.startDatabaseServer(DatabaseManagementServiceFactory.java:288) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.graphdb.facade.DatabaseManagementServiceFactory.build(DatabaseManagementServiceFactory.java:223) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.server.CommunityBootstrapper.createNeo(CommunityBootstrapper.java:38) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.server.NeoBootstrapper.start(NeoBootstrapper.java:185) [neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.server.NeoBootstrapper.start(NeoBootstrapper.java:100) [neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.server.CommunityEntryPoint.main(CommunityEntryPoint.java:30) [neo4j-5.26.0.jar:5.26.0]
neo4j-1  | Caused by: org.neo4j.kernel.lifecycle.LifecycleException: Component 'org.neo4j.kernel.extension.GlobalExtensions@7906578e' failed to initialize. Please see the attached cause exception "io.prometheus.client.Collector".
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.init(LifeSupport.java:334) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.init(LifeSupport.java:56) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:85) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.graphdb.facade.DatabaseManagementServiceFactory.startDatabaseServer(DatabaseManagementServiceFactory.java:279) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      ... 5 more
neo4j-1  | Caused by: org.neo4j.kernel.extension.FailedToBuildExtensionException: Failed to build kernel extension Extension:GraphDataScienceExtension[gds.closed] because it is compiled with a reference to a class, method, or field, that is not in the class path: 'io/prometheus/client/Collector'. The most common cause of this problem, is that Neo4j has been upgraded without also upgrading all installed extensions, such as APOC. Make sure that all of your extensions are build against your specific version of Neo4j.
neo4j-1  |      at org.neo4j.kernel.extension.ExtensionFailureStrategies.wrap(ExtensionFailureStrategies.java:56) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.extension.ExtensionFailureStrategies$1.handle(ExtensionFailureStrategies.java:68) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.extension.AbstractExtensions.init(AbstractExtensions.java:68) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.init(LifeSupport.java:317) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.init(LifeSupport.java:56) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:85) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.graphdb.facade.DatabaseManagementServiceFactory.startDatabaseServer(DatabaseManagementServiceFactory.java:279) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      ... 5 more
neo4j-1  | Caused by: java.lang.NoClassDefFoundError: io/prometheus/client/Collector
neo4j-1  |      at com.neo4j.gds.metrics.GdsMetrics.create(GdsMetrics.java:33) ~[graph-data-science.jar:?]
neo4j-1  |      at com.neo4j.gds.extension.GraphDataScienceAndExtrasExtensionBuilder.create(GraphDataScienceAndExtrasExtensionBuilder.java:134) ~[graph-data-science.jar:?]
neo4j-1  |      at com.neo4j.gds.extension.GraphDataScienceExtension.newInstance(GraphDataScienceExtension.java:72) ~[graph-data-science.jar:?]
neo4j-1  |      at com.neo4j.gds.extension.GraphDataScienceExtension.newInstance(GraphDataScienceExtension.java:37) ~[graph-data-science.jar:?]
neo4j-1  |      at org.neo4j.kernel.extension.AbstractExtensions.newInstance(AbstractExtensions.java:124) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.extension.AbstractExtensions.init(AbstractExtensions.java:62) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.init(LifeSupport.java:317) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.init(LifeSupport.java:56) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:85) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.graphdb.facade.DatabaseManagementServiceFactory.startDatabaseServer(DatabaseManagementServiceFactory.java:279) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      ... 5 more
neo4j-1  | Caused by: java.lang.ClassNotFoundException: io.prometheus.client.Collector
neo4j-1  |      at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
neo4j-1  |      at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
neo4j-1  |      at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
neo4j-1  |      at com.neo4j.gds.metrics.GdsMetrics.create(GdsMetrics.java:33) ~[graph-data-science.jar:?]
neo4j-1  |      at com.neo4j.gds.extension.GraphDataScienceAndExtrasExtensionBuilder.create(GraphDataScienceAndExtrasExtensionBuilder.java:134) ~[graph-data-science.jar:?]
neo4j-1  |      at com.neo4j.gds.extension.GraphDataScienceExtension.newInstance(GraphDataScienceExtension.java:72) ~[graph-data-science.jar:?]
neo4j-1  |      at com.neo4j.gds.extension.GraphDataScienceExtension.newInstance(GraphDataScienceExtension.java:37) ~[graph-data-science.jar:?]
neo4j-1  |      at org.neo4j.kernel.extension.AbstractExtensions.newInstance(AbstractExtensions.java:124) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.extension.AbstractExtensions.init(AbstractExtensions.java:62) ~[neo4j-kernel-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.init(LifeSupport.java:317) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.init(LifeSupport.java:56) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:85) ~[neo4j-common-5.26.0.jar:5.26.0]
neo4j-1  |      at org.neo4j.graphdb.facade.DatabaseManagementServiceFactory.startDatabaseServer(DatabaseManagementServiceFactory.java:279) ~[neo4j-5.26.0.jar:5.26.0]
neo4j-1  |      ... 5 more
neo4j-1  | 2024-12-12 13:24:56.885+0000 INFO  Neo4j Server shutdown initiated by request
neo4j-1  | 2024-12-12 13:24:56.885+0000 INFO  Stopped.

```

**Comment by WindyOrgUK** [comment]
Hi @AnikinNN 

Please see
https://github.com/neo4j/graph-data-science/issues/340

**Comment by AnikinNN** [comment]
@WindyOrgUK  
Seems like the same issue, thank you for connecting them

**Comment by Mats-SX** [comment]
@AnikinNN Thank you for reporting this issue. I am pleased to announce the release of GDS version 2.13.2, which should make your docker setup green again. Please report back here again if it still does not work for you, and we'll reopen the issue.

All the best
Mats

**Comment by AnikinNN** [comment]
Looks like issue solved in GDS version 2.13.2
